tools/zim_update

```diff
@@ -2,10 +2,35 @@
 # zim_update - update the zim repository
 #
 
+echo "\nUpdating Zim"
 cd ${ZIM_HOME}
 
 # this is the cleanest way I know how to update a repository
 git remote update -p
 git merge --ff-only @\{u\}
 # and update the submodules
 git submodule update --init --recursive
```

Comment:
this is unnecessary as there are no submodules left :)

Reply:
this is so people can still manage their modules using submodules if they so chose to.